blackbox.yml

```diff
@@ -3,6 +3,10 @@ modules:
     prober: http
     timeout: 5s
     http:
+      method: POST
```

Comment:
This file is the default config, it shouldn't be changed for tests.

I'd put what you need in blackbox-good.yml